app/models/concerns/host_searching.rb

```diff
@@ -32,6 +32,12 @@ module HostSearching
                   only_explicit: true, operators: ['=']
     scoped_search on: :stale_timestamp,
                   only_explicit: true, operators: ['<', '>']
+    scoped_search on: :supported,
```

Comment:
To be more specific, can it be named `ssg_supported`? `supported` sounds very generic.